pages/contributors.tsx

```diff
@@ -7,41 +7,64 @@ import { Spinner } from "react-bootstrap"
 import Lottie from "react-lottie"
 import Card from "../components/Cards/Card"
 
-const defaultOptions = {
+type ContributorSerializer = {
+    handle_name:string
+    name:string
+    handle_url:string
+    image:string
+    github:string
+    description:string[]
+}
+
+type ContriResponseSerializer = {
+    login:string
+    url:string
+    avatar_url:string
+    html_url:string
+}
+
+interface DefOptType {
+    loop: boolean;
+    autoplay: boolean;
+    animationData:any;
+  }
+
+const defaultOptions: DefOptType = {
     loop: true,
     autoplay: true,
     animationData: animationData,
 }
 
 export default function Contributors() {
-    const [contriFrontend, setContributorsFront] = useState()
-    const [contriBackend, setContributorsBack] = useState()
-    const [fullname, setFullName] = useState([{}])
-    const [contributors, setContributors] = useState()
-    const [error, setError] = useState(false)
+    const [contriFrontend, setContributorsFront] = useState<ContriResponseSerializer[]>(null)
+    const [contriBackend, setContributorsBack] = useState<ContriResponseSerializer[]>(null)
+    const [fullname, setFullName] = useState<Object[]>([{}] as Object[])
```

Comment:
Not sure about this part, but can we make it more specific than saying it to be an object? Like it'd be better if we know the properties of data objects in this array.

Reply:
problem is that the keys are not already known

Reply:
No problem then.